internal/captain/command.go

```diff
@@ -27,6 +29,22 @@ import (
 	"github.com/ActiveState/cli/internal/table"
 )
 
+// appEventPrefix is used for all executables except for the State Tool itself.
+var appEventPrefix string = func() string {
+	cmdName, err := os.Executable()
+	if err != nil {
+		return "unknown"
+	}
```

Comment:
Let's fall back on `os.Args[0]` instead.

Reply:
This is done, and I also moved the logic to osutils. It seems like it would be useful in other places. Please let me know if you'd like it back in the captain package.